include/mbedtls/x509_crt.h

```diff
@@ -651,4 +651,9 @@ int mbedtls_x509write_crt_pem( mbedtls_x509write_cert *ctx, unsigned char *buf,
 }
 #endif
 
+#ifndef MBEDTLS_TLS_DEFAULT_ALLOW_SHA1
+/* The test infrastructure requires a positive define */
+#define MBEDTLS_X509__DEFAULT_FORBID_SHA1
+#endif
```

Comment:
I am not very happy about this change, but I understand the rationale behind it. Ideally, we should extend our test infrastructure to fulfil this requirement. @sbutcher-arm could you please comment?